src/diffusers/pipelines/alt_diffusion/pipeline_alt_diffusion.py

```diff
@@ -425,17 +428,25 @@ def _encode_prompt(
 
         return prompt_embeds
 
-    def run_safety_checker(self, image, device, dtype):
-        if self.safety_checker is not None:
-            safety_checker_input = self.feature_extractor(self.numpy_to_pil(image), return_tensors="pt").to(device)
+    def run_safety_checker(self, image, device, dtype, output_type="pil"):
+        if self.safety_checker is None or output_type == "latent":
+            has_nsfw_concept = False
```

Comment:
I _think_ this should always be a list, see another comment on that. Not sure if we could break existing workflows though.